language/changes/5-named-addresses.md

```diff
@@ -0,0 +1,212 @@
+# Named Addresses
+
+- Status: Implemented in Move 1.3
+
+## Introduction
+
+Named addresses are a new source language only feature (that is, the named address feature is
+compiled away and not present in the Move bytecode format). The feature allows names to be used in
+place of numerical values in any spot where addresses are used. Named addresses are declared as top
+level elements (outside of modules and scripts) with `address MyAddr;` and can be assigned like so
+`address MyAddr = 0x42;`
+
+With the landing of this feature, Move standard library modules now reside under the address `Std`
+and must be used as such, e.g. `Std::Vector`. Similarly, diem framework modules now reside under the
```

Comment:
I think the "must be used as such" behavior is worth explaining above, perhaps a clarification that this is doing something stronger than creating an alias between 0x42 and `MyAddr` would be enough.

Reply:
Also, nit: diem framework -> Diem Framework

Reply:
Added a bit more on this, since it is a breaking change of sorts